### PR TITLE
Schema and List output made default

### PR DESCRIPTION
**Pull request by breillyr**

### CODE REVIEW

File: dockpulp/cli.py [hunk 1]
```diff
@@ -614,10 +614,10 @@ def do_list(bopts, bargs, parser):
                       help='show labels')
     parser.add_option('-m', '--manifests', default=False, action='store_true',
                       help='only list manifests and their tags, no blobs')
-    parser.add_option('--schema', default=False, action='store_true',
-                      help='display schema version for each manifest')
-    parser.add_option('--lists', default=False, action='store_true',
-                      help="display manifest lists")
```
Comment (twaugh) [inline dockpulp/cli.py]: Does that mean scripts using `--schema` or `--lists` will cause errors?

Comment (breillyr) [reply dockpulp/cli.py]: That's a good point. I should probably reroute those options to dummy variables and outputting a deprecation warning...

Comment (breillyr) [reply dockpulp/cli.py]: @twaugh made the change, thanks

File: tests/test_cli.py [hunk 2]
```diff
@@ -321,7 +321,7 @@ def test_do_empty(self, mocked_pulp, bargs):
     @patch('dockpulp.Pulp')
     def test_do_list(self, mocked_pulp, caplog, silent):
         bopts = testbOpts()
-        bargs = ['test-repo', '--content', '--details', '--labels', '--lists']
+        bargs = ['test-repo', '--content', '--details', '--labels']
```
Comment (twaugh) [inline tests/test_cli.py]: Could undo this change now, to check the deprecated options still work.
